### ISSUE TITLE
[BUG] Header profile button fails WCAG 2.2 2.4.4 Link Purpose

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#157]
**Describe the bug**
The button text using the users name does not clearly identify the purpose of the button and fails WCAG Success Criterion 2.4.4 Link Purpose, Level A.

**Expected behavior**
The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, except where the purpose of the link would be ambiguous to users in general. 

My thoughts are Profile or Account could work or use an Icon with with a aria-label of Profile or Account. The icon used when the page is zoomed in could be used with an aria-label of Profile or Account.
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/c2b284e6-7dfb-440d-8c86-72a1ee5bca47)

**Screenshots**
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/ab213492-2e9f-4c27-b87a-64a0635248d8)

**Comment by ecstaticrainbow** [comment]
needs aria-labels, describing what the button is actually doing

**Comment by KatiePUX** [comment]
Tickets 114, 157, 178, 149 linked - let team Kai know when you are working on these so they can make adjustments when required

**Comment by buzzub** [comment]
In PR #254
